Cargo.toml

```diff
@@ -67,8 +69,8 @@ ctrlc = { version = "3.4", features = ["termination"] }
 cargo-husky = { version = "1", default-features = false, features = ["run-for-all", "prepush-hook", "run-cargo-fmt"] }
 cargo-audit = "0.18"
 tracing-subscriber = { version ="0.3", features = ["env-filter"] }
-hyper-proxy = "0.9"
-hyper = { version ="0.14", features = ["full"] }
+hyper-proxy2 = "0.1"
```

Comment:
Published [hyper-proxy2](https://github.com/siketyan/hyper-proxy2) as a fork of hyper-proxy, which is not maintained.

Reply:
@siketyan wow, that's very kind of you.
It was only used in examples and was in [dev-dependencies]. We can just remove it, since this is not our scope :)

Reply:
@abdolence So you mean to remove entire the proxy_client example?

Reply:
Removed at [e7067bd](https://github.com/abdolence/slack-morphism-rust/pull/232/commits/e7067bd544290fbc92dff01e0450c7d0edc158e4).

Reply:
I'm ok to leave now, since you have already migrated it :) I'm just saying if it blocks you we can get rid of it.

Reply:
okay, rolled back for users who needs proxy

Reply:
@siketyan please return it back :D you've done some job already, let's have it. I'm fine with `hyper-proxy2`